### ISSUE TITLE
test files

### ISSUE THREAD

**Issue by skhiggins** [skhiggins/PythonTools#32]
Improvements to the newest test files:

- [x] They include a line that is no longer necessary:
```python
urlbase = 'http://www.inegi.org.mx'  #change this line whenever you change the website to scrape
```

- [x] It is good coding practice to make the arguments clear and only include unnamed arguments in the first one or two arguments of the function. Also include spaces after function arguments. So:
```python
get_files(url_base,'pdf',folder_base,True)
```
should instead be:
```python
get_files(url_base, 'pdf', folder=folder_base, overwrite=True)
```

- [x] Make the names of files consistent: `<command name>_test_<description>.ipynb` (without the <> symbols) -- right now some of them are `<command name>_test` and some are `test_<command name>`. Also let's try to be more concise in the `<description>` part of the file name (i.e. a few words) so they aren't so long, and make sure they don't contain spaces in the name. Then in the file itself, you can include some description at the top in writing so it's clear what this file is doing (note you can change a cell to have its input be Markdown instead of code which is good for describing what you are doing in the file at the top).

**Comment by skhiggins** [comment]
Also:
- [ ] Make the names of files consistent: `<command name>_test_<description>.ipynb` (without the <> symbols) -- right now some of them are '<command name>_test` and some are `test_<command name>`. Also let's try to be more concise in the `<description>` part of the file name so they aren't so long, and make sure they don't contain spaces in the name.

**Comment by angelynaye** [comment]
Solved in #35
